tests/foreman/virtwho/test_cli_virtwho.py

```diff
@@ -244,16 +248,17 @@ def test_positive_hypervisor_id_option(self):
         for value in values:
             VirtWhoConfig.update({'id': vhd['id'], 'hypervisor-id': value})
             result = VirtWhoConfig.info({'id': vhd['id']})
-            self.assertEquals(result['connection']['hypervisor-id'], value)
+            self.assertEqual(result['connection']['hypervisor-id'], value)
             config_file = get_configure_file(vhd['id'])
             command = get_configure_command(vhd['id'])
             deploy_configure_by_command(command)
-            self.assertEquals(
+            self.assertEqual(
                 get_configure_option('hypervisor_id', config_file), value)
         VirtWhoConfig.delete({'name': name})
         self.assertFalse(VirtWhoConfig.exists(search=('name', name)))
 
-    @tier1
+    @skip_if_bug_open('bugzilla', 1735670)
```

Comment:
The bug is already ON_QA status, if it can be verified by manual, we should not skip this case.

Reply:
Thanks for the reminder, yes we can test it now, I will debug this and remove the flag if the bug is verified